src/Generators/Webserver/Database/Drivers/PostgreSQL.php

```diff
@@ -129,6 +129,10 @@ protected function flushConnection(IlluminateConnection $connection, array $conf
 
     protected function dropPriviliges(IlluminateConnection $connection, array $config)
     {
+        if (config("tenancy.db.tenant-division-mode") === Connection::DIVISION_MODE_SEPARATE_SCHEMA) {
+            return true;
+        }
```

Comment:
Can you give a little insight into why we should not do this?

Reply:
In my use-case I have only one database user, that is the main database user used on the System connection as well as all other Tenant connections. Removing privileges for a user would remove all the items they owned, regardless of schema they were in. That is dangerous, and in my case broke functionality.